esphome/components/sen5x/sen5x.cpp

```diff
@@ -135,9 +135,12 @@ void SEN5XComponent::setup() {
       ESP_LOGD(TAG, "Firmware version %d", this->firmware_version_);
 
       if (this->voc_sensor_ && this->store_baseline_) {
-        // Hash with compilation time
+        uint32_t combined_serial = (uint32_t(this->serial_number_[0]) << 24) |
+                                   (uint32_t(this->serial_number_[1]) << 16) | (uint32_t(this->serial_number_[2]));
```

Comment:
This probably needs formatting and also `#include "esphome/core"helpers.h"
```suggestion
        uint32_t combined_serial = encode_uint24(this->serial_number_[0], this->serial_number_[1], this->serial_number_[2]);
```

Reply:
Thanks!